packages/lesswrong/lib/betas.ts

```diff
@@ -42,6 +42,8 @@ export const userHasShortformTags = isEAForum ? shippedFeature : disabled;
 
 export const userHasCommentProfileImages = disabled;
 
+export const userHasEagProfileImport = disabled;
```

Comment:
What's going on with this? Link me to a slack discussion?

Reply:
We decided to hide it because it hasn't been migrated to work with salesforce yet (see the end of [this thread](https://cea-core.slack.com/archives/C03G0AQ925P/p1683127662377709)).

Note that the page still exists if you know the URL - it just isn't linked to from the account settings page.